src/doc/trpl/arrays-vectors-and-slices.md

```diff
@@ -49,7 +49,7 @@ languages.
 
 A *vector* is a dynamic or "growable" array, implemented as the standard
 library type [`Vec<T>`](../std/vec/) (we'll talk about what the `<T>` means
-later). Vectors always allocate their data on the heap. Vectors are to slices 
+later). Vectors always allocate their data on the heap. Vectors are to arrays
```

Comment:
Ahh interesting. The intention with this section is that a vector is a  owning data structure. A slice is a borrowing data structure. A String is an owning data structure. A string slice is a borrowing data structure. Internally, String is a wrapper over a Vec, and &'str is basically the same thing as a slice, a pointer and a length.